deployment/terraform/create.go

```diff
@@ -613,6 +638,34 @@ func (t *Terraform) setupElasticSearchServer(extAgent *ssh.ExtAgent, ip string)
 	}
 }
 
+// waitForGreenCluster blocks until the cluster's status is green or the
+// provided timeout is reached
+func waitForGreenCluster(dur time.Duration, es *elasticsearch.Client) error {
+	timeout := time.After(dur)
+	for {
+		select {
+		case <-timeout:
+			return fmt.Errorf("timeout after %s, cluster's status did not become green", dur.String())
+		case <-time.After(30 * time.Second):
+			clusterHealth, err := es.ClusterHealth()
```

Comment:
Is there a timeout set for this request? Just want to make sure this couldn't block indefinitely or something.

Reply:
It defaults to 30 seconds, yes: https://www.elastic.co/guide/en/elasticsearch/reference/7.10/cluster-health.html